### ISSUE TITLE
[Sourcing][Accessibility][fdp-table] Continuous reading issue while focusing on table row

### ISSUE THREAD

**Issue by yixuan-liu** [SAP/fundamental-ngx#10870]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
When using reader to focus on any particular table row, the whole row information including all the cell content is being read in one go where users cannot understand the information and the placements of the cells which is being read.

When focus on a row, should read both column label and cell content. The content should be correctly conveyed by announcing its associated column header followed by the actual content.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
- Angular 15
- F-NGX Platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using a simple basic table example https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table, and use screen reader to focus on any particular table row.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Label, information, properties of all UI elements should be conveyed correctly once to the screen reader users when accessed and additional reading should be avoided.

**Comment by rvamudha** [comment]
@yixuan-liu - this is P2 can you please tag this with [P2]

**Comment by droshev** [comment]
The recommendation is: remove the complete row focus so we won't need to read the row info.
